### DISCUSSION TITLE
Super stream creation using Java Client API

### DISCUSSION THREAD

**Discussion by vinodreddyb** [rabbitmq/rabbitmq-stream-java-client#403] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

As per documentation super stream can be created using CLI 

rabbitmq-streams add_super_stream invoices --partitions 3

But how do I create same with Java Client library. I don't find any example in documentation. 

### Describe the solution you'd like

Provide any example to create super stream from java API

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ.

Just so you know, if a GitHub repository has both "Issues" and "Discussions", you should use "Discussions" for questions. Issues are for actual bugs.

**Comment by acogoluegnes** [comment]
It's not possible with the stream protocol, we may introduce this feature later though. In the meantime, it is still possible to use plain AMQP, here is how it is done with the AMQP Java client in the test suite:

https://github.com/rabbitmq/rabbitmq-stream-java-client/blob/4fa7e81a76e614fe8000d546ffc6ddf5aee2ab68/src/test/java/com/rabbitmq/stream/impl/TestUtils.java#L284-L321

**Reply by acogoluegnes** [reply, marked as answer]
It is now possible to create and delete super streams with the stream protocol, and the stream Java client exposes these commands.
